internal/pkg/cli/task_run.go

```diff
@@ -121,7 +127,7 @@ func newTaskRunOpts(vars runTaskVars) (*runTaskOpts, error) {
 		sel:     selector.NewSelect(vars.prompt, store),
 		spinner: termprogress.NewSpinner(),
 
-		deployer: cloudformation.New(sess),
+		deployer:      cloudformation.New(sess),
 	}
 
 	opts.configureRuntimeOpts = func() error {
```

Comment:
We could possibly define `func() error {` this function outside since this is very big as an anonymous function.